src/components/pricing/FeatureComparisonTable.js

```diff
@@ -0,0 +1,263 @@
+import React, { Fragment } from 'react';
+import { CheckIcon, MinusIcon } from '@heroicons/react/solid';
+import { TextLink } from 'components';
+
+const TIERS = [{
+  name: 'Teams',
+  ctaLabel: 'Start a free trial',
+  ctaLinkTo: '/free-trial/',
+}, {
+  name: 'Growth',
+  ctaLabel: 'Request a demo',
+  ctaLinkTo: '/request-demo/',
+}];
+
+const sections = [{
+  name: 'Scale',
+  features: [{
+    name: 'Software components tracked',
+    tiers: { Teams: 'Unlimited', Growth: 'Unlimited' },
+  }, {
+    name: 'API Specs',
+    tiers: { Teams: 'Unlimited', Growth: 'Unlimited' },
+  }, {
+    name: 'TechDocs',
+    tiers: { Teams: 'Unlimited', Growth: 'Unlimited' },
+  }, {
+    name: 'Scaffolder templates',
+    tiers: { Teams: 'Unlimited', Growth: 'Unlimited' },
+  }, {
+    name: 'Minimum seats',
```

Comment:
This line indicates that we prevent small companies from using the Roadie API or custom plugins since they won't be buying enough seats

Reply:
Addressed in a standup last week.